torchvision/transforms/functional_tensor.py

```diff
@@ -344,7 +344,9 @@ def _blend(img1: Tensor, img2: Tensor, ratio: float) -> Tensor:
     return (ratio * img1 + (1 - ratio) * img2).clamp(0, bound).to(img1.dtype)
 
 
-def _rgb2hsv(img):
+def _rgb2hsv(img: Tensor) -> Tensor:
+    if not isinstance(img, torch.Tensor):
```

Comment:
`_rgb2hsv` is a private method, I think we can skip type checking and raising error here.
Btw, `functional_tensor.py` module is intended to be private too, user should not use those functions in the code. Warnings will be added soon according to https://github.com/pytorch/vision/issues/2547